src/AsyncStatusIndicator/AsyncStatusIndicator.tsx

```diff
@@ -27,16 +52,16 @@ export const Props = {
  */
 @skinnable()
```

Comment:
I would remove @skinnable completely from brc as we've been doing in AlinityPRO and GDSM